tests/unit/core/binarycodec/test_main.py

```diff
@@ -253,7 +253,7 @@ def test_xaddress_null_tag(self):
         self.assertEqual(encode(json_null_x), encode(json_null_r))
 
     def test_xaddress_invalid(self):
-        with self.assertRaises(ValueError):
+        with self.assertRaises(XRPLBinaryCodecException):
```

Comment:
This is technically a bugfix - it is better for this error to raise `XRPLBinaryCodecException` for this than to raise `ValueError`